src/test/java/com/zendesk/maxwell/ColumnDefTest.java

```diff
@@ -172,7 +175,7 @@ public void TestDate() {
 		assertTrue(d.matchesMysqlType(MySQLConstants.TYPE_DATE));
 
 		Date date = new GregorianCalendar(1979, 10, 1).getTime();
-		assertThat(d.toSQL(date), is("1979-10-01"));
+		assertThat(d.toSQL(date), is("''1979-11-01''"));
```

Comment:
single single-quotes I think -- the premise is to get the value back to a string that can be put directly into SQL.